2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'project': 'meme_experiments', 'entity': 'vietnguyen', 'root_dir': '/content/SSLMemes', 'run_id': 'gysqirq7', 'sweep_param_path': '/content/SSLMemes/wandb/sweep-uhu1kilc/config-gysqirq7.yaml', 'sweep_id': 'uhu1kilc'}
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'src/train.py', 'program': '/content/SSLMemes/src/train.py'}
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_init.py:_log_setup():405] Logging user logs to /content/SSLMemes/wandb/run-20221011_042754-gysqirq7/logs/debug.log
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_init.py:_log_setup():406] Logging internal logs to /content/SSLMemes/wandb/run-20221011_042754-gysqirq7/logs/debug-internal.log
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_init.py:init():439] calling init triggers
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_init.py:init():443] wandb.init called with sweep_config: {'batch_size': 40, 'dropout': 0.3, 'epochs': 1, 'hidden_size': 128, 'learning_rate': 0.07194015231258541, 'optimizer': 'sgd'}
config: {}
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [wandb_init.py:init():492] starting backend
2022-10-11 04:27:54,254 INFO    Thread-7  :28710 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-10-11 04:27:54,263 INFO    Thread-7  :28710 [backend.py:ensure_launched():219] starting backend process...
2022-10-11 04:27:54,386 INFO    Thread-7  :28710 [backend.py:ensure_launched():225] started backend process with pid: 28853
2022-10-11 04:27:54,397 INFO    Thread-7  :28710 [wandb_init.py:init():501] backend started and connected
2022-10-11 04:27:54,408 INFO    Thread-7  :28710 [wandb_run.py:_config_callback():992] config_cb None None {'batch_size': 40, 'dropout': 0.3, 'epochs': 1, 'hidden_size': 128, 'learning_rate': 0.07194015231258541, 'optimizer': 'sgd'}
2022-10-11 04:27:54,415 INFO    Thread-7  :28710 [wandb_init.py:init():565] updated telemetry
2022-10-11 04:27:54,554 INFO    Thread-7  :28710 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-10-11 04:27:55,499 INFO    Thread-7  :28710 [wandb_run.py:_on_init():1759] communicating current version
2022-10-11 04:27:55,540 INFO    Thread-7  :28710 [wandb_run.py:_on_init():1763] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-11 04:27:55,540 INFO    Thread-7  :28710 [wandb_init.py:init():625] starting run threads in backend
2022-10-11 04:27:58,219 INFO    Thread-7  :28710 [wandb_run.py:_console_start():1733] atexit reg
2022-10-11 04:27:58,220 INFO    Thread-7  :28710 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-10-11 04:27:58,220 INFO    Thread-7  :28710 [wandb_run.py:_redirect():1611] Redirecting console.
2022-10-11 04:27:58,221 ERROR   Thread-7  :28710 [wandb_run.py:_redirect():1670] Failed to redirect.
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/wandb_run.py", line 1663, in _redirect
    out_redir.install()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/usr/local/lib/python3.7/dist-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib/python3.7/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-10-11 04:27:58,221 INFO    Thread-7  :28710 [wandb_init.py:init():664] run started, returning control to user process
2022-10-11 04:27:58,222 INFO    Thread-7  :28710 [wandb_run.py:_config_callback():992] config_cb None None {'img_feature_path': 'data/features/visualbert/', 'train_csv_path': 'data/splits/random/memotion_train.csv', 'val_csv_path': 'data/splits/random/memotion_val.csv', 'model_type': 'visualbert', 'model_path': 'uclanlp/visualbert-nlvr2-coco-pre', 'eval_step': 100, 'amp': True, 'output_dir': './tmp', 'checkpoint_step': 100, 'random_seed': 42, 'resume_training': False, 'semi_supervised': True}
2022-10-11 04:27:58,597 INFO    Thread-7  :28710 [wandb_run.py:_config_callback():992] config_cb None None {'output_hidden': 768}
2022-10-11 04:28:00,129 INFO    Thread-7  :28710 [wandb_run.py:finish():1539] finishing run vietnguyen/meme_experiments/gysqirq7
2022-10-11 04:28:00,129 INFO    Thread-7  :28710 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 1
2022-10-11 04:28:00,131 INFO    Thread-7  :28710 [wandb_run.py:_restore():1674] restore
2022-10-11 04:28:00,137 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 04:28:00,349 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 699
}

2022-10-11 04:28:00,513 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 8780
}

2022-10-11 04:28:00,615 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 699
  total_bytes: 10100
}

2022-10-11 04:28:00,717 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10100
  total_bytes: 10100
}

2022-10-11 04:28:00,818 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10100
  total_bytes: 10100
}

2022-10-11 04:28:00,920 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10100
  total_bytes: 10100
}

2022-10-11 04:28:01,022 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10100
  total_bytes: 10100
}

2022-10-11 04:28:01,123 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10100
  total_bytes: 10100
}

2022-10-11 04:28:01,629 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10100
  total_bytes: 10100
}

2022-10-11 04:28:01,991 INFO    Thread-7  :28710 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 10100
  total_bytes: 10100
}
local_info {
}

2022-10-11 04:28:03,550 INFO    Thread-7  :28710 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-10-11 04:28:03,550 INFO    Thread-7  :28710 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-10-11 04:28:03,552 INFO    Thread-7  :28710 [wandb_run.py:_footer_sync_info():2822] logging synced files
